.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "es5",
+  "semi": false
+}
```

Comment:
Can you move this into the package.json pls?

Reply:
There's already a prettier config in there. https://github.com/infinitered/ChainReactApp2023/blob/main/package.json#L126-L131